appservice/src/tree/AppSettingTreeItem.ts

```diff
@@ -14,22 +14,26 @@ export class AppSettingTreeItem extends AzureTreeItem<ISiteTreeRoot> {
     public static contextValue: string = 'applicationSettingItem';
     public readonly contextValue: string = AppSettingTreeItem.contextValue;
     public readonly parent: AppSettingsTreeItem;
+    public readonly commandLabel: string | undefined;
 
-    private key: string;
-    private value: string;
+    private _key: string;
+    private _value: string;
+    private _hideValue: boolean;
 
     constructor(parent: AppSettingsTreeItem, key: string, value: string) {
         super(parent);
-        this.key = key;
-        this.value = value;
+        this._key = key;
+        this._value = value;
+        this.commandId = 'appService.toggleValueVisability';
```

Comment:
You'll have to pass this command Id in. Just like the problem we had with deployments node commandId